core/store/src/trie/mod.rs

```diff
@@ -103,6 +101,55 @@ pub enum KeyLookupMode {
     MemOrTrie,
 }
 
+#[derive(Clone, Copy, Debug)]
```

Comment:
I know this is not a heavy struct to copy but curious why we need to derive `Copy` and not just use `.clone()`?

Reply:
We don't *need* to do so, in fact I could avoid implementing `Clone` either and instead opt to reconstruct `AccessOptions { ..opts }` everywhere, where sharing was necessary. However this for all intents and purposes is just a shared reference, and that is `Copy` so there is no reason to not make this `Copy` either.